src/BuildScriptGenerator/DotNetCore/DotNetCorePlatformInstaller.cs

```diff
@@ -31,5 +33,22 @@ public virtual bool IsVersionAlreadyInstalled(string version)
                 dynamicInstallDir: Path.Combine(
                     this.CommonOptions.DynamicInstallRootDir, DotNetCoreConstants.PlatformName));
         }
+
+        public override void InstallPlatformSpecificSkeletonDependencies(StringBuilder stringBuilder)
+        {
+            stringBuilder.AppendLine($"echo 'Installing {DotNetCoreConstants.PlatformName} specific dependencies...'");
+
+            // .NET Core dependencies (this is universal for all versions of .NET Core)
+            stringBuilder.AppendAptGetInstallPackages(
+                "libc6",
+                "libgcc1",
+                "libgssapi-krb5-2",
+                "libstdc++6",
+                "zlib1g",
+                "libuuid1",
+                "libunwind8");
+
+            InstallPythonToolingAndLanguage(stringBuilder);
```

Comment:
Wondering why we also install python tools here, does it require python tools in .NET apps?

Reply:
hmmm, could've sworn I responded to this a few days ago. Essentially though, we need to install python for dotnet and node due to some use cases (e.g., Blazor).